backend/report_submission/views.py

```diff
@@ -363,6 +388,10 @@ def get(self, request, *args, **kwargs):
                 "DB_id": SN[NC.ADDITIONAL_EINS].snake_case,
                 "instructions_url": instructions_base_url + "additional-eins-workbook/",
                 "workbook_url": workbook_base_url + "additional-eins-workbook.xlsx",
+                # below URL handled as a special case because of inconsistent name usage in audit/urls.py and audit/cross_validation/naming.py
+                "existing_workbook_url": reverse(
+                    "audit:AdditionalEins", args=[report_id]
+                ),
```

Comment:
@tadhg-ohiggins - had to take `AdditionalEins`, `AdditionalUeis` and `FederalAwardsExpended` as special cases here because the naming/casing in `FORM_SECTIONS` (which drives `urls.py`) doesn't match the `camel_case` entries in `naming.py` - flagging for visibility